test/built-ins/Symbol/species/Symbol.species.in_Map.js

```diff
@@ -0,0 +1,18 @@
+// Copyright 2015 Cubane Canada, Inc.  All rights reserved.
+// See LICENSE for details.
+
+/*---
+info: >
+ Map has a property at `Symbol.species`
+es6id: 23.1.2.2
+author: Sam Mikes
+description: Map[Symbol.species] exists per spec
```

Comment:
Probably could stand to add the features array for this one so implementations without map can skip (although not super important)